diff --git a/node_modules/react-native-background-timer/android/build.gradle b/node_modules/react-native-background-timer/android/build.gradle
index 863d198..f8dfc95 100644
--- a/node_modules/react-native-background-timer/android/build.gradle
+++ b/node_modules/react-native-background-timer/android/build.gradle
@@ -5,14 +5,25 @@ def safeExtGet(prop, fallback) {
 }
 
 android {
-    compileSdkVersion safeExtGet('compileSdkVersion', 28)
+    namespace 'com.ocetnik.timer'
+    compileSdkVersion safeExtGet('compileSdkVersion', 36)
     
     defaultConfig {
         minSdkVersion safeExtGet('minSdkVersion', 16)
-        targetSdkVersion safeExtGet('targetSdkVersion', 28)
+        targetSdkVersion safeExtGet('targetSdkVersion', 36)
         versionCode 1
         versionName "1.0"
     }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_17
+        targetCompatibility JavaVersion.VERSION_17
+    }
+}
+
+repositories {
+    google()
+    mavenCentral()
 }
 
 dependencies {
